applications/app/controllers/AllIndexController.scala

```diff
@@ -15,15 +15,21 @@ import LiveContentApi.getResponse
 object AllIndexController extends Controller with ExecutionContexts with ItemResponses with Dates with Logging {
 
   // no need to set the zone here, it gets it from the date.
-  private val dateFormat = DateTimeFormat.forPattern("yyyy/MMM/dd").withZone(Edition.defaultEdition.timezone)
   private val dateFormatUTC = DateTimeFormat.forPattern("yyyy/MMM/dd").withZone(DateTimeZone.UTC)
 
-  def newer(path: String, day: String, month: String, year: String) = Action.async{ implicit request =>
-    val edition = Edition(request)
-    val requestedDate: DateTime = dateFormat.withZone(edition.timezone).parseDateTime(s"$year/$month/$day")
-    findNewer(path, requestedDate).map{ _.map{ date =>
-      val foundDate = date.withZone(edition.timezone)
-      Found(s"/$path/${urlFormat(foundDate)}/all")
+  private def requestedDate(dateString: String) = {
+    dateFormatUTC
+      .parseDateTime(dateString)
+      .withTimeAtStartOfDay()
+      .plusDays(1)
+      .minusSeconds(1)
+      .toDateTime
+  }
+
+  def altDate(path: String, day: String, month: String, year: String) = Action.async{ implicit request =>
+    val reqDate = requestedDate(s"$year/$month/$day").withTimeAtStartOfDay()
```

Comment:
I'd be tempted to not called `requestedDate` method here but instead just inline the methods used as it makes a bit simpler to read. So if I have read it right it will be 
```
 val reqDate = dateFormatUTC.parseDateTime(s"$year/$month/$day").withTimeAtStartOfDay()
```

?